--- a/cmake/toolchain-riscv64-linux-musl-x86_64.cmake
+++ b/cmake/toolchain-riscv64-linux-musl-x86_64.cmake
@@ -5,7 +5,7 @@ include(CMakeForceCompiler)
 set( CMAKE_SYSTEM_NAME          Linux )
 set( CMAKE_SYSTEM_PROCESSOR     riscv )
 set( ARCH riscv )
-set( CROSS_COMPILE riscv64-unknown-linux-musl- )
+set( CROSS_COMPILE riscv64-unknown-linux-gnu- )
 
 set(CMAKE_C_COMPILER ${CROSS_COMPILE}gcc)
 set(CMAKE_CXX_COMPILER ${CROSS_COMPILE}g++)
---