diff --git a/components/sophgo/video/CMakeLists.txt b/components/sophgo/video/CMakeLists.txt
index a3982f7..aa792b6 100644
--- a/components/sophgo/video/CMakeLists.txt
+++ b/components/sophgo/video/CMakeLists.txt
@@ -24,7 +24,7 @@ set(VIDEO_RREQIRDS
     ${SG200X_SDK_PATH}/cvi_mpi/lib/libvi.a
     ${SG200X_SDK_PATH}/cvi_mpi/lib/libvo.a
     ${SG200X_SDK_PATH}/cvi_mpi/lib/libcvi_ispd2.a
-    json-c
+    ${SG200X_SDK_PATH}/install/soc_sg2002_recamera_emmc/tpu_musl_riscv64/cvitek_tpu_sdk/lib/libcvi_json-c.a
     ${SG200X_SDK_PATH}/cvi_mpi/lib/libsns_ov5647.a
     ${SG200X_SDK_PATH}/cvi_mpi/lib/libsns_sc530ai_2l.a
 )
