diff --git a/components/sscma-micro/CMakeLists.txt b/components/sscma-micro/CMakeLists.txt
index 15cb40e..925bc83 100644
--- a/components/sscma-micro/CMakeLists.txt
+++ b/components/sscma-micro/CMakeLists.txt
@@ -39,5 +39,5 @@ component_register(
     COMPONENT_NAME sscma-micro
     SRCS ${SOURCES}
     INCLUDE_DIRS ${INCS}
-    PRIVATE_REQUIREDS  mosquitto ssl crypto sophgo cares hv cvi_rtsp ${SG200X_SDK_PATH}/install/soc_sg2002_recamera_emmc/tpu_musl_riscv64/cvitek_tpu_sdk/lib/libcviruntime-static.a ${SG200X_SDK_PATH}/install/soc_sg2002_recamera_emmc/tpu_musl_riscv64/cvitek_tpu_sdk/lib/libcvikernel-static.a
+    PRIVATE_REQUIREDS  mosquitto ssl crypto sophgo cares hv ${SG200X_SDK_PATH}/install/soc_sg2002_recamera_emmc/tpu_musl_riscv64/cvitek_tpu_sdk/lib/libcviruntime-static.a ${SG200X_SDK_PATH}/install/soc_sg2002_recamera_emmc/tpu_musl_riscv64/cvitek_tpu_sdk/lib/libcvikernel-static.a
 ) 
\ No newline at end of file
