diff --git a/Makefile b/Makefile
index 598674e..6b0066e 100644
--- a/Makefile
+++ b/Makefile
@@ -143,6 +143,10 @@ deps-y+=$(libsbi-objs-path-y:.o=.dep)
 deps-y+=$(libsbiutils-objs-path-y:.o=.dep)
 deps-y+=$(firmware-objs-path-y:.o=.dep)
 
+define CHIP_ARCH
+CV181X
+endef
+
 ifeq (,$(wildcard ${PM_SRAM_BIN_PATH}))
 	ifeq (${CHIP_ARCH},CV180X)
 		PM_SRAM_BIN_PATH=${OPENSBI_PATH}/pm_default_cv180x.bin
@@ -229,7 +237,7 @@ GENFLAGS	+=	$(libsbiutils-genflags-y)
 $(eval $(call add_define,CONFIG_OPENSBI_FASTBOOT))
 endif
 
-CFLAGS		=	-g -Wall -Werror -ffreestanding -nostdlib -fno-strict-aliasing -O2
+CFLAGS		=	-g -Wall -Werror -ffreestanding -nostdlib -fno-stack-protector -fno-strict-aliasing -O2
 CFLAGS		+=	-fno-omit-frame-pointer -fno-optimize-sibling-calls
 CFLAGS		+=	-mno-save-restore -mstrict-align
 CFLAGS		+=	-mabi=$(PLATFORM_RISCV_ABI) -march=$(PLATFORM_RISCV_ISA)
